proxylib/testparsers/headerparser.policy

```diff
@@ -0,0 +1,38 @@
+[{
+    "endpointSelector": {"matchLabels":{"id.echoserver":""}},
+    "ingress": [{
+        "fromEndpoints": [
+            {"matchLabels":{"reserved:host":""}},
+            {"matchLabels":{"id.client":""}}
+        ],
+        "toPorts": [{
+            "ports": [{"port": "2701", "protocol": "tcp"}],
+            "rules": {
+                "l7proto": "test.headerparser",
+                "l7": [{
+                    "rule": {
+                        "prefix": "foo"
+                    }
+                }]
+            }
+        }]
+    }]
+},{
+    "endpointSelector": {"matchLabels":{"id.echoserver":""}},
+    "ingress": [{
+        "fromEndpoints": [
+            {"matchLabels":{"reserved:host":""}},
+            {"matchLabels":{"id.client":""}}
+        ],
+        "toPorts": [{
+            "ports": [{"port": "2701", "protocol": "tcp"}],
+            "rules": {
+                "l7proto": "test.headerparser",
+                "l7": [
+		    {"rule": {"prefix": "bar", "contains": "beer"}},
```

Comment:
I was afraid of this. Do we need to repeat  ourselves when adding a rule? It seems tow word `"rule"` repeats multiple times. Can we avoid it? Or is the intention to have more things other than "rule"?